docile/tools/dataset_browser.py

```diff
@@ -336,9 +339,13 @@ def get_all_displayboxes(self) -> List[DisplayBox]:
                     ]
                 )
         else:
-            fields_types.extend(
-                [(f, DisplayType.ANNOTATION) for f in annotation.page_fields(self.page_i)]
-            )
+            try:
+                fields_types.extend(
+                    [(f, DisplayType.ANNOTATION) for f in annotation.page_fields(self.page_i)]
+                )
+            except KeyError:
+                # annotations not available, this can happen for test set or unlabeled set
```

Comment:
How about adding a logging message here?

Reply:
I tried that first but it was quite annoying because it pulled apart (by a few red rows) the dataset browser buttons and the opened page (on every page).